### ISSUE TITLE
File not found error when moving files

### ISSUE THREAD

**Issue by kane-liu** [garriguv/danger-ruby-swiftformat#20]
When moving files like
```
A/xxx.swift → B/xxx.swift
```
`check_format` failed.

```
[!] Invalid `Dangerfile` file: Error running SwiftFormat:  
Error: �[0mRunning SwiftFormat...  
(dryrun mode - no files will be changed.)  
error: File not found at /Users/distiller/project/A/xxx.swift.. Updating the Danger gem might fix the issue. Your Danger version: 6.0.6, latest Danger version: 6.0.8
```
Updating Danger version: 6.0.8 can not solve this error.

**Comment by bawjensen** [comment]
So this is affecting us as well, and I did some digging. It seems to be a bug on quite a few different levels, from [weird behavior with SwiftFormat](https://github.com/nicklockwood/SwiftFormat/issues/467) to https://github.com/ruby-git/ruby-git (maybe incorrectly?) flagging renamed files with the `:type => 'modified'`.

However, I believe this is a bug that can be fixed at multiple places in the stack, and any of them are valid, so I'll open a PR with a proposed solution within this plugin.

**Comment by garriguv** [comment]
The fix is merged and I've released `0.6.0`, we should be good now.

Thank you for the fix, @bawjensen !
